Gemfile.lock

```diff
@@ -315,6 +307,9 @@ GEM
       mustache
       nokogiri
     libxml-ruby (3.2.0)
+    listen (3.3.0)
```

Comment:
The change does not reflect the version mentioned in the PR.
Should be `3.3.1` rather  `3.3.0`